samples/CoreAnimationTest/CoreAnimationTest.vsimporter/CoreAnimationTest-WinStore10/CoreAnimationTest-Debug-xcvars.txt

```diff
@@ -87,15 +87,15 @@ PRODUCT_BUNDLE_IDENTIFIER = com.Microsoft.CoreAnimationTest
 PRODUCT_NAME = CoreAnimationTest
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = CoreAnimationTest
-PROJECT_DIR = E:/GIT_REPOS/winobjc/samples/CoreAnimationTest
-PROJECT_FILE_PATH = E:\GIT_REPOS\winobjc\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
+PROJECT_DIR = Y:/Arcadia/WinObjC/samples/CoreAnimationTest
+PROJECT_FILE_PATH = Y:\Arcadia\WinObjC\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
```

Comment:
@asimonov-msft Why are PROJECT_DIR, PROJECT_FILE_PATH, SOURCE_ROOT, and SRCROOT even needed in this file? Can we get rid of them altogether?